src/sentry/static/sentry/app/views/settings/components/forms/richListField.jsx

```diff
@@ -0,0 +1,231 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {t} from 'app/locale';
+import DropdownAutoComplete from 'app/components/dropdownAutoComplete';
+import DropdownButton from 'app/components/dropdownButton';
+import InputField from 'app/views/settings/components/forms/inputField';
+import InlineSvg from 'app/components/inlineSvg';
+import Confirm from 'app/components/confirm';
+
+const ItemList = styled('div')`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const Item = styled('span')`
```

Comment:
Why span and not a div?

Reply:
Thinking about it, this should actually be a `<li>`.